bin/docker-trace.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+COMMAND=$1
+if [ $COMMAND = 'start' ]; then
+  docker run -p 127.0.0.1:27017:27017 -d mongo
+  docker run -p 127.0.0.1:6379:6379 -d redis
+  docker run -p 127.0.0.1:3306:3306 -e MYSQL_ROOT_PASSWORD='Password12!' -e MYSQL_DATABASE=test -d mysql
+fi
+if [ $COMMAND = 'stop' ]; then
+  docker stop `docker ps | sed -En 's/([a-f0-9]+)[[:space:]]+(redis|mongo|mysql).*/\1/p'`
```

Comment:
Another option would be to use `--name trace-test-mongo`, `--name trace-test-redis` etc. when using `docker run`.  Then the command 
```
docker stop `docker ps -q --filter name="trace-test-*"`
```
could be used to stop everything started for trace testing.  This would avoid stopping a `redis`, `mongo`, or `mysql` container that was started for a purpose other than trace testing.